gdarchen-kannelle/kannelle-test-web
# Add auth0 identification

# 🎯 Goal
Add auth0 in order to be more practical to visualize a charter


# 🧠 Approach
Same approach as that used for the web dashboard.
Reorganization of resources into a specific component

# 🧪 Test

- Load the page `http://localhost:3000`
- You should be redirected to the auth0 authentication page
- Log In
- Change the charter and visualize the result
- Log out
- Log in with another account
- Change the charter and visualize the result

# 📚 Notion Card
 - [InternShip - Add auth0 on Kannelle test](https://www.notion.so/kannelle/InternShip-Add-auth0-on-Kannelle-test-dbaae4b3b5f14e3f8cc95325cb9ada09)

# ✅ Definition of Done

Check the following conditions once they are met (write it like `[x]`):

- [x] The title of the pull request describes it sufficiently and is formatted like `Subject - PR Goal`
- [x] I selected the _reviewer_ of the pull request
- [x] I assigned myself as the _assignee_ of the pull request
- [x] I configured the _labels_ describing the context of the pull request (`bugfix`, `feature`, `docs`, etc.)
- [x] I removed the descriptions and the examples in the sections above
